tests/foreman/ui/test_branding.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for branding
+
+:Requirement: Branding
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Branding
+
+:Assignee: akjha
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.config import settings
+
+
+def test_check_version_login_screen(default_sat):
+    """Check correct version on the login screen
+
+    :id: f48110ad-29b4-49b1-972a-a70075a05732
+
+    :Steps: Curl the login screen
+
+    :expectedresults: Correct version depending on the snap
+
+    :CaseLevel: System
+
+    :BZ: 1315849, 1367495, 1372436, 1502098, 1540710, 1582476, 1724738, 1959135, 2076979
+    """
+    result = default_sat.execute(
+        f'curl -k {default_sat.url}/users/login | '
+        f'grep -e "LoginPage" -e "{settings.server.version.release}"'
+    )
+    assert 'LoginPage' in result.stdout
+    assert str(settings.server.version.release) in result.stdout
```

Comment:
Considering that you already grepped these values in your command, asserting again isn't necessary here, right?

Reply:
These are just simple assertions. Sure, not necessary. But just being explicit here.

Reply:
Consistency is a good thing, so I vote to keep the assert. (If forced to choose, I would rather drop `-e "{settings.server.version.release}` and keep the assert line; but thats just my opinion)

Reply:
I think its a lot better to assert this against `default_sat.version` instead of the settings value.

Reply:
Agreed with @swadeley here, this isn't a very strong assertion and is basically validating that `grep` works.

Reply:
new changes. Resolving this.